### PR TITLE
Add support for ReverseV2 operator

### PR DESCRIPTION
**Pull request by buddhapuneeth**
Reverse V2 is not an ONNX operator, so it is translated into Reverse Sequence and other set of valid ONNX operators.

### CODE REVIEW

**Comment by buddhapuneeth** [comment]
Can one of you check why few checks are failing? Batch_norm test is failing, which is not related to my changes and unable to reproduce locally.

**Comment by jiafatom** [comment]
> Can one of you check why few checks are failing? Batch_norm test is failing, which is not related to my changes and unable to reproduce locally.

Just checked there is one test failure -- random number fails -- Sometimes it happens. Could you make a trivial change to your PR and resubmit again? It should be fine.

File: keras2onnx/_builtin.py [hunk 1]
```diff
@@ -2211,6 +2211,61 @@ def convert_tf_zeros_like(scope, operator, container):
                               name=operator.full_name)
 
 
+@converter_func(TYPES.ReverseV2)
+def convert_tf_reverse(scope, operator, container):
+    node = operator.raw_operator
+    oopb = OnnxOperatorBuilder(container, scope)
+    ip = node.inputs[0]
+    axes = _cal_tensor_value(node.inputs[1])
+    if len(_cal_tensor_shape(axes)) != 1:
```
Comment (jiafatom) [inline keras2onnx/_builtin.py:2220]: Combine these two if?

Comment (buddhapuneeth) [reply keras2onnx/_builtin.py]: done

**Pull request merged**